crates/uv/tests/it/python_install.rs

```diff
@@ -3950,3 +3952,180 @@ fn python_install_upgrade_version_file() {
     hint: The version request came from a `.python-version` file; change the patch version in the file to upgrade instead
     ");
 }
+
+#[test]
+fn python_install_compile_bytecode() -> anyhow::Result<()> {
+    let context: TestContext = TestContext::new_with_versions(&[])
+        .with_filtered_python_keys()
+        .with_filtered_exe_suffix()
+        .with_managed_python_dirs()
+        .with_empty_python_install_mirror()
+        .with_python_download_cache();
+
+    // Install 3.14 and compile its bytecode
+    uv_snapshot!(context.filters(), context.python_install().arg("--compile-bytecode").arg("3.14"), @r"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Bytecode compiled 1052 files for cpython-3.14.2-[PLATFORM] in [TIME]
+    Installed Python 3.14.2 in [TIME]
+     + cpython-3.14.2-[PLATFORM] (python3.14)
+    ");
+
+    // Find the stdlib path for cpython 3.14
+    let stdlib = fs_err::read_link(
+        context
+            .bin_dir
+            .child(format!("python3.14{}", std::env::consts::EXE_SUFFIX)),
+    )?
+    .parent()
+    .context("Python binary should be a child of `bin`")?
+    .parent()
+    .context("`bin` directory should be a child of the installation path")?
+    .join("lib")
+    .join("python3.14");
+
+    // And the count should match
+    let count = {
+        let mut count = 0;
+        let walker = WalkDir::new(stdlib).into_iter();
+        for entry in walker {
+            let entry = entry?;
+            let path = entry.path();
+            if entry.metadata()?.is_file() && path.extension().is_some_and(|ext| ext == "pyc") {
+                count += 1;
+            }
+        }
+        count
+    };
+
+    insta::assert_snapshot!(count.to_string(), @"1052");
```

Comment:
Doesn't that count change when we bump the patch version?

Reply:
Although there are still the logs, or should I add a filter?